clients/fides-js/src/lib/preferences.ts

```diff
@@ -220,6 +220,16 @@ export const updateConsent = async (
     );
   }
 
+  const handleValidationError = (errorMessage: string) => {
+    if (validation === "throw") {
```

Comment:
can these be enums?

Reply:
good suggestion! I'll create a task in Jira for that so we don't hold up the release any longer. :)